src/main/webapp/css/recipe-page.css

```diff
@@ -0,0 +1,33 @@
+.list-header {
+  border-bottom: 5px solid #5386A2;
+  border-radius: 5px;
+  left: 10px;
+  position: relative;
+  width: min-content;
+}
+
+#image-description {
+  align-items: flex-start;
+  display: flex;
+  left: 10px;
+  position:relative;
+}
+
+#description {
+  font-size: 20px;
+  margin: 0px 15px 5px 10px;
```

Comment:
super nit: you don't need units on 0. 0px -> 0

Reply:
Ack and fixed